frontend/aml_dashboard/src/index.js

```diff
@@ -0,0 +1,746 @@
+  }
+
+  const n_job = n_jobs.$value.get().toString();
+  const n_iter = features.parameters['Iterations'].value;
+  const percentage_data = features.parameters['Percentage of data'].value;
+
+  fetch(url_train + n_job + '/' + n_iter + '/' + percentage_data, {
+    method: "POST", // *GET, POST, PUT, DELETE, etc.
+    mode: "cors", // no-cors, *cors, same-origin
+    cache: "no-cache", // *default, no-cache, reload, force-cache, only-if-cached
+    credentials: "omit", // include, *same-origin, omit
+    headers: {
+      "Content-Type": "application/json",
+    },
+    redirect: "follow", // manual, *follow, error
+    referrerPolicy: "no-referrer", // no-referrer, *no-referrer-when-downgrade, origin, origin-when-cross-origin, same-origin, strict-origin, strict-origin-when-cross-origin, unsafe-url
+    body: JSON.stringify(json_dt), // body data type must match "Content-Type" header
+  })
+  .then(response => response.json())
+  .then(async json => {
+    console.log(json);
+    console.log('RECEIVED');
+
+    save_to_file(json, "model_.json");
+
+    aml_model['ready'] = true;
+
+    textAMLTrainStatus.$value.set('<h1>Finished :)</h1> ');
+
+  });
+});
+
+const params = modelParameters(classifier);
+const prog = trainingProgress(classifier);
+const plotTraining = trainingPlot(classifier);
+
+
+// -----------------------------------------------------------
+// FETCHING
+// ----------------------------------------------------------
+
+const url_fetcher = "http://localhost:5000/fetcher/";
+
+const statistics_received = text('No statistics received');
+statistics_received.title = 'AML Statistics';
+
+const model_received = text('No model received');
+model_received.title = 'AML Model';
+
+const collaborative_status = text('Not received');
+collaborative_status.title = 'AML Collaborative Learning Status';
+
+const search_statistics = button('Search for statistics');
+search_statistics.title = 'AML Statistics Fetcher'
+
+search_statistics.$click.subscribe( async () => {
+  console.log('Search statistics');
+  collaborative_status.$value.set(' <h1>Searching statistics...</h1> <br> <img src="https://i.gifer.com/Cad.gif"> ');
+
+  fetch(url_fetcher + 'statistics', {
+    method: "GET", // *GET, POST, PUT, DELETE, etc.
+    mode: "cors", // no-cors, *cors, same-origin
+    cache: "no-cache", // *default, no-cache, reload, force-cache, only-if-cached
+    credentials: "omit", // include, *same-origin, omit
+    headers: {
+      "Content-Type": "application/json",
+    },
+    redirect: "follow", // manual, *follow, error
+    referrerPolicy: "no-referrer", // no-referrer, *no-referrer-when-downgrade, origin, origin-when-cross-origin, same-origin, strict-origin, strict-origin-when-cross-origin, unsafe-url
+  })
+  .then(response => response.json())
+  .then(async json => {
+    console.log(json);
+    console.log('STATISTICS RECEIVED');
+
+    const name = json.name; // Extracting the value of 'name' field from the JSON response
+
```

Comment:
Please take into account the case when statistics may not be found.
```suggestion
    collaborative_status.$value.set('<h1>Statistics not found !</h1> ');
    statistics_received.$value.set('<h2>Statistics not found'</h2>');
```

Reply:
Addressed in #2